src/app/common/pipes/dynamicFilterPipe.ts

```diff
@@ -0,0 +1,30 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'dynamicFilter'
+})
+export class DynamicFilterPipe implements PipeTransform {
+
+  transform(items: any[], filter: any): any[] {
```

Comment:
`any` is not a nice type (and it's thus quite unclear from what to what this transforms)